odoo/fields.py

```diff
@@ -769,6 +761,10 @@ def make_domain(path, model):
     _related_groups = property(attrgetter('groups'))
     _related_aggregator = property(attrgetter('aggregator'))
 
+    @lazy_property
+    def column_type(self):
```

Comment:
Please add a call to `lazy_property.reset_all(self)` inside method `setup`.

Reply:
Actually this isn't useful by design, because we never re-setup a given field.
When overridden, a new instance of `Field` is created, and so on.